xbmc/cores/dvdplayer/DVDDemuxers/DVDDemuxVobsub.h

```diff
@@ -35,7 +35,7 @@ class CDVDDemuxVobsub : public CDVDDemux
   CDVDDemuxVobsub();
   virtual ~CDVDDemuxVobsub();
 
-  virtual bool          Open(const std::string& filename, const std::string& subfilename = "");
+  virtual bool          Open(const std::string& filename, int source, const std::string& subfilename = "");
```

Comment:
right, then let's keep this. maybe drop the virtual and move it down to make clear that this is no overwrite of the base class' method?